spec/factories/user.rb

```diff
@@ -19,5 +18,15 @@
     confirmed_at { Time.zone.now }
     created_at { Time.zone.now }
     updated_at { Time.zone.now }
+
+    transient do
```

Comment:
Would probably remove this same as the method above, not sure it's still a useful attribute now?

Reply:
Oh I see, you're just using it as the input for the after_create, :+1:

Reply:
Yes, this is still useful. We still have a `old_user_id` column on users for now because we need to make sure all the associations are created before we can remove it.

This transient value means that we don't have to update tests yet :)